core/includes/update.inc

```diff
@@ -294,6 +294,88 @@ function update_module_add_to_system($modules = array()) {
   }
 }
 
+/** 
+ * Checks for disabled dependencies during a Drupal 7 upgrade.
+ * 
+ * @return string
+ *    A themed message that lists modules that will be enabled,
+ *    to be displayed in the 'info' page of the update process. 
+ */
+
+function update_upgrade_check_dependencies() {
+  $status_report = '';
+  if (backdrop_get_installed_schema_version('system') > 7000) {
+    $modules_to_enable = update_upgrade_modules_to_enable();
+    
+    if (empty($modules_to_enable)) {
+      return;
```

Comment:
```suggestion
      return '';
```
Since this function should return a string.

Reply:
Thanks @bugfolder. I will include this in the next PR in a minute.